scss/_utilities.scss

```diff
@@ -71,6 +71,7 @@
 .text-lowercase      { text-transform: lowercase !important; }
 .text-uppercase      { text-transform: uppercase !important; }
 .text-capitalize     { text-transform: capitalize !important; }
+.text-ucfirst        { &:first-letter { text-transform: uppercase; } }
```

Comment:
The pseudo-class form is deprecated and all of our supported browsers support the newer pseudo-element form.